### PR TITLE
Amazon Redshift revamp

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Revamped Amazon Redshift source doc and moved Models topic inside Warehouse Actions.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: warehouse-actions/amazon-redshift.mdx [hunk 1]
```diff
@@ -95,93 +102,86 @@ ALTER DEFAULT PRIVILEGES IN SCHEMA "<YOUR_SCHEMA>" GRANT SELECT ON TABLES TO rud
 Run the above command only if you're okay with RudderStack being able to access the data in all the future tables residing within your specified schema.
 </div>
 
-## Setting Up the Source
-
-To set up Amazon Redshift as a source in RudderStack, follow these steps:
-
-* Log into your [RudderStack dashboard](https://app.rudderlabs.com/signup?type=freetrial).
-* From the left panel, select **Sources**. Then, click on **Add Source**, as shown:
+**Replace `<YOUR_SCHEMA>` with the exact name of your Redshift schema.**
```
Comment (ashishRudder) [inline warehouse-actions/amazon-redshift.mdx:105]: replace `<YOUR_SCHEMA>` and "<YOUR_TABLE>"

File: warehouse-actions/amazon-redshift.mdx [hunk 2]
```diff
@@ -95,93 +102,86 @@ ALTER DEFAULT PRIVILEGES IN SCHEMA "<YOUR_SCHEMA>" GRANT SELECT ON TABLES TO rud
...
+<div class="infoBlock">
 
-If you've already configured Redshift as a source before, your existing credentials will automatically appear under **Use Existing Credentials**.
+For more information on the difference between the <strong>Table</strong> and <strong>Model</strong> options when creating a Warehouse Actions source, refer to the <a href="#faq">FAQ</a> section below.
 </div>
 
-* Next, enter the relevant connection details in the **Connection Credentials** as shown below:
-
-<!--![](../.gitbook/assets/screen-shot-2021-01-06-at-3.59.08-pm.png)-->
+2. Enter the relevant settings in the **Connection Credentials** section as listed below:
```
Comment (ashishRudder) [inline warehouse-actions/amazon-redshift.mdx:129]: This is true only if Table is selected.

Comment (ashishRudder) [reply warehouse-actions/amazon-redshift.mdx:129]: basically this `2.` step can be skipped.

File: warehouse-actions/amazon-redshift.mdx [hunk 3]
```diff
@@ -95,93 +102,86 @@ ALTER DEFAULT PRIVILEGES IN SCHEMA "<YOUR_SCHEMA>" GRANT SELECT ON TABLES TO rud
...
+<div class=“infoBlock”>
+  
+If you have already configured a destination in RudderStack, choose the <strong>Use Existing Destination</strong> option which will take you to the <strong>Schema</strong> tab in the source settings. To add a new destination from scratch, select the <strong>Create New Destination</strong> option which will take you to the destination configuration page.
+</div>
 
-<!--![](../.gitbook/assets/screen-shot-2021-01-06-at-4.06.07-pm.png)-->
+## Specifying the data to import
 
-<img src="../assets/screen-shot-2021-01-06-at-4.06.07-pm.png" alt="Connect Destinations" />
+While connecting a destination to your Warehouse Actions source, you can use the default JSON mapping or the [Visual Data Mapping](https://www.rudderstack.com/docs/warehouse-actions/features/visual-data-mapper/) feature to map the warehouse columns to your destination fields.
```
Comment (ashishRudder) [inline warehouse-actions/amazon-redshift.mdx:175]: re: `feature to map the warehouse columns to your destination fields` is not factually correct for `JSON mapping`

**Review by ashishRudder — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**